release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -444,6 +444,11 @@ In {product-title} {product-version}, the installation program uses the Cluster
 [id="ocp-release-notes-networking_{context}"]
 === Networking
 
+[id="ocp-4-19-networking-support-load-secrets_{context}"]
+==== Creating a route with externally managed certificate (General Availability) 
+
+With this release, {product-title} routes can be configured with third-party certificate management solutions, utilising the `.spec.tls.externalCertificate` field in the route API. This allows you to reference externally managed TLS certificates through secrets, streamlining the process by eliminating manual certificate management. By using externally managed certificates, you reduce errors, ensure a smoother certificate update process, and enable the OpenShift router to promptly serve renewed certificates. For more information, see xref:../../networking/index#nw-ingress-route-secret-load-external-cert_secured-routes[Creating a route with externally managed certificate].
```

Comment:
This xref isn't creating a link, likely because of the missing `.adoc` at the end of the file:
```suggestion
With this release, {product-title} routes can be configured with third-party certificate management solutions, utilising the `.spec.tls.externalCertificate` field in the route API. This allows you to reference externally managed TLS certificates through secrets, streamlining the process by eliminating manual certificate management. By using externally managed certificates, you reduce errors, ensure a smoother certificate update process, and enable the OpenShift router to promptly serve renewed certificates. For more information, see xref:../../networking/index.adoc#nw-ingress-route-secret-load-external-cert_secured-routes[Creating a route with externally managed certificate].
```